<?xml version="1.0" encoding="UTF-8"?><testrun name="AutherServiceTest" project="jpa-library-project" tests="2" started="2" failures="1" errors="0" ignored="0">
  <testsuite name="com.example.book.service.AutherServiceTest" time="0.199" displayname="AutherServiceTest" uniqueid="[engine:junit-jupiter]/[class:com.example.book.service.AutherServiceTest]">
    <testcase name="findByEmailNotFoundServiceTest" classname="com.example.book.service.AutherServiceTest" time="0.175" displayname="findByEmailNotFoundServiceTest()" uniqueid="[engine:junit-jupiter]/[class:com.example.book.service.AutherServiceTest]/[method:findByEmailNotFoundServiceTest()]">
      <failure>
        <expected>false</expected>
        <actual>true</actual>
        org.opentest4j.AssertionFailedError: expected: &lt;false&gt; but was: &lt;true&gt;&#13;
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)&#13;
	at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)&#13;
	at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)&#13;
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)&#13;
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:177)&#13;
	at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:1145)&#13;
	at com.example.book.service.AutherServiceTest.findByEmailNotFoundServiceTest(AutherServiceTest.java:30)&#13;
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)&#13;
&#13;

      </failure>
    </testcase>
    <testcase name="findByEmailFoundServiceTest" classname="com.example.book.service.AutherServiceTest" time="0.022" displayname="findByEmailFoundServiceTest()" uniqueid="[engine:junit-jupiter]/[class:com.example.book.service.AutherServiceTest]/[method:findByEmailFoundServiceTest()]"/>
  </testsuite>
</testrun>
